src/DaisyFx/ServiceCollectionExtensions.cs

```diff
@@ -1,20 +1,23 @@
-﻿using System;
+﻿using DaisyFx;
 using DaisyFx.Events;
 using DaisyFx.Hosting;
 using DaisyFx.Sources.Http;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Routing;
 using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
+using System;
 
-namespace DaisyFx
+// ReSharper disable CheckNamespace
+// ReSharper disable UnusedType.Global
+// ReSharper disable MemberCanBePrivate.Global
+namespace Microsoft.Extensions.DependencyInjection
 {
-    public static class DaisyExtensions
+    public static class ServiceCollectionExtensions
```

Comment:
Following the same logic, shouldn't [MapDaisy](https://github.com/DaisyFx/DaisyFx/blob/6b65e19b36c71532955a4574a5ee11a2427859a0/src/DaisyFx/ServiceCollectionExtensions.cs#L44) be placed in `Microsoft.AspNetCore.Routing` in a `EndpointRouteBuilderExtensions` since it doesn't have any relationship with ServiceCollection

Reply:
True. Will update the PR soon.

Reply:
Moved it into `Microsoft.AspNetCore.Builder` namespace. Noticed other `.Map` methods using this namespace instead and to minimize to many usings.